buf.gen.ui.yaml

```diff
@@ -0,0 +1,18 @@
+# buf.gen.yaml defines a local generation template.
+# For details, see https://docs.buf.build/configuration/v1/buf-gen-yaml
+version: v1
+plugins:
+  - remote: buf.build/bufbuild/plugins/es:v0.1.1-1
+    out: public/gen
+    opt: target=ts
+  - remote: buf.build/bufbuild/plugins/connect-web:v0.2.1-1
+    out: public/gen
```

Comment:
Is there a reason we should have both `public/gen` and `public/proto`?

Reply:
The `_pb.js` files look very different if recompiled with all pointing to `public/proto`. It is a bit weird.